clang/lib/Sema/CheckedCAnalysesPrepass.cpp

```diff
@@ -70,16 +82,17 @@ class PrepassHelper : public RecursiveASTVisitor<PrepassHelper> {
       return T->getAsRecordDecl();
     }
 
-    // AddBoundsSiblingField adds FieldWithBounds to the set of fields in
-    // whose declared bounds F occurs.
-    void AddBoundsSiblingField(const FieldDecl *F) {
-      auto It = Info.BoundsSiblingFields.find(F);
-      if (It != Info.BoundsSiblingFields.end())
-        It->second.insert(FieldWithBounds);
+    // AddPtrWithBounds adds PtrWithBounds to the set of variables in whose
```

Comment:
Nit: could this comment say something like "...to the set of variables or fields" or "...to the set of decls" since this method can also be used to update `BoundsSiblingFields`?

Reply:
Yes, I though about adding "variables or fields" to the comment. But currently it updates variables and fields but in future it may be invoked on something else. So I wanted didn't want to keep extending the comment. That's the reason I simply went with variables (because I think it is general enough).